### PR TITLE
Create Web Vitals collection

### PR DESCRIPTION
**Pull request by khempenius**
- Adds a Web Vitals collection at web.dev/learn-web-vitals. URL is `/learn-web-vitals` in order to clearly distinguish it from `/vitals`.
- Switches `/vitals` to the standard `post` layout.

cc @philipwalton

### CODE REVIEW

**Comment by robdodson** [comment]
I think you could also delete https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/page.njk because vitals was the only one using it.

**Comment by robdodson** [comment]
It looks like the vitals graphic disappeared from the homepage. You may need to update it in _data/paths/vitals.json?

**Review by robdodson — APPROVED** [review]

**Comment by robdodson** [comment]
Everything looks good to me. @kaycebasques @philipwalton can we get an LGTM from y'all?

**Comment by philipwalton** [comment]
Can we hold off on merging until we resolve the issue (being discussed via email) of whether or not the web.dev/vitals page should continue to use a custom template or move to a blog post template?

**Comment by philipwalton** [comment]
Also, I just noticed that the staged version of the /vitals page has two "Give feedback" forms:
https://deploy-preview-3899--web-dev-staging.netlify.app/vitals/

**Review by kaycebasques — CHANGES_REQUESTED** [review]
Marking as **Request changes** so that we don't forget that we need to discuss stuff more.

File: src/site/_data/paths/learn-web-vitals.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "learn-web-vitals",
+    "cover": "/images/collections/web-vitals.svg",
+    "title": "Web Vitals",
+    "description": "Key signals of quality user experiences.",
```
Comment (philipwalton) [inline src/site/_data/paths/learn-web-vitals.json:5]: Change this to "Essential metrics for a healthy site" to keep it consistent with the current landing page.

File: src/site/_data/paths/learn-web-vitals.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "overview": "Web Vitals define the performance metrics and measurements that indicate a delightful user experience.",
```
Comment (philipwalton) [inline src/site/_data/paths/learn-web-vitals.json:6]: Probably best to reuse the same language from the web.dev/vitals intro content: "An initiative to provide unified guidance for quality signals that are essential to delivering a great user experience on the web."

Side note: I've always found that language to be a bit awkward, so if we want to change it we should probably change it in both places.

File: src/site/_data/paths/learn-web-vitals.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            "title": "Web Vitals Metrics",
+            "pathItems": [
+                "lcp",
+                "fid",
+                "cls",
+                "defining-core-web-vitals-thresholds"
+            ]
+        },
+        {
+            "title": "Measuring Web Vitals",
```
Comment (philipwalton) [inline src/site/_data/paths/learn-web-vitals.json:24]: Per the [style guide](https://web.dev/handbook/grammar/#titles-and-headings), I think this needs to be "Measure Web Vitals"

File: src/site/_data/paths/learn-web-vitals.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            "pathItems": [
+                "vitals-measurement-getting-started",
+                "vitals-tools",
+                "vitals-field-measurement-best-practices"
+            ]
+        },
+        {
+            "title": "Improving Web Vitals",
```
Comment (philipwalton) [inline src/site/_data/paths/learn-web-vitals.json:32]: Same comment as above.

**Review by philipwalton — CHANGES_REQUESTED** [review]

File: src/site/content/en/vitals/index.md [hunk 5]
```diff
@@ -1,13 +1,12 @@
 ---
-layout: page
+layout: post
 title: Web Vitals
-subhead: Essential metrics for a healthy site
 description: Essential metrics for a healthy site
+hero: web-vitals-hero.png
 authors:
   - philipwalton
```
Comment (kaycebasques) [inline src/site/content/en/vitals/index.md:7]: @philipwalton we should remove your author attribution here? It makes the page seem more authoritative.

**Review by kaycebasques — CHANGES_REQUESTED** [review]
Seems like a step in the right direction! I've got a question for @philipwalton.

**Review by MichaelSolati — APPROVED** [review]

**Review by MichaelSolati — APPROVED** [review]
I'm sure there are a few tweaks that need to happen still, but as far as the data structure LGTM.

File: src/site/content/en/vitals/index.md (hunk 5 above)
Comment (philipwalton) [reply src/site/content/en/vitals/index.md:7]: I go back and forth on this. The benefit of attribution is readers know who to reach out to for questions, but I agree it does make it look a bit more like a blog post (though I'm not sure how much of that is the author image vs. the overall layout).

**Review by kaycebasques — APPROVED** [review]
Seems like we're good to go here

**Comment by kaycebasques** [comment]
Oh @philipwalton this is blocked on your LGTM

**Review by philipwalton — APPROVED** [review]

**Pull request merged**